src/main/java/net/imagej/table/BoolColumn.java

```diff
@@ -66,4 +66,31 @@ public void setHeader(final String header) {
 		return Boolean.class;
 	}
 
+	@Override
+	public void fill(final Boolean[] values) {
```

Comment:
I suppose I'll see why you need this in later commits, but... it sucks to ever have an array of the wrapped primitive types. We want to use `boolean[]` over `Boolean[]` whenever possible...